csharp/Worker/Microsoft.Spark.CSharp/Worker.cs

```diff
@@ -343,12 +343,20 @@ private static IFormatter ProcessCommand(Stream inputStream, Stream outputStream
                         continue;
                     }
 
-                    WriteOutput(outputStream, serializerMode, message, formatter);
+                    try
+                    {
+                        WriteOutput(outputStream, serializerMode, message, formatter);
+                    }
+                    catch (Exception)
+                    {
+                        logger.LogError("WriteOutput() failed at iteration {0}", count);
+                        throw;
+                    }
                     count++;
                     funcProcessWatch.Start();
                 }
 
-                logger.LogDebug("Output entries count: " + count);
+                logger.LogInfo("Output entries count: " + count);
```

Comment:
Prompt the log level here, which give us information about the output entries. By default, LogDebug is not shown in the log.